kmod/src/xattr.c

```diff
@@ -666,6 +732,11 @@ static int scoutfs_xattr_set(struct dentry *dentry, const char *name,
 		goto unlock;
 	}
 
+	if (tgs.worm && scoutfs_inode_worm_denied(inode)) {
```

Comment:
We don't want the tgs.worm qualifier here, we want to forbid all xattr changes if _denied is true.